extension/docs/HowToRun.md

```diff
@@ -18,21 +18,61 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-## Environment variables
+## Environment variable
```

Comment:
could you ensure the links are not dead, also please update TOCs.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/542/commits/c250367e6a21f15824b20a49509417a3d6587ebc